src/index.js

```diff
@@ -54,16 +56,6 @@ module.exports =
 
             const flowConfig = find(file, '.flowconfig')
             if (!flowConfig) {
-              if (!that.lastConfigError[file] ||
```

Comment:
I want to keep this.
I'm willing to change the frequency, but the errors are important.